web/react/components/more_direct_channels.jsx

```diff
@@ -3,67 +3,102 @@
 
 var ChannelStore = require('../stores/channel_store.jsx');
 var TeamStore = require('../stores/team_store.jsx');
+var Client = require('../utils/client.jsx');
+var AsyncClient = require('../utils/async_client.jsx');
 var utils = require('../utils/utils.jsx');
 
 module.exports = React.createClass({
+    displayName: 'MoreDirectChannels',
     componentDidMount: function() {
         var self = this;
-        $(this.refs.modal.getDOMNode()).on('show.bs.modal', function(e) {
+        $(this.refs.modal.getDOMNode()).on('show.bs.modal', function showModal(e) {
             var button = e.relatedTarget;
-            self.setState({ channels: $(button).data('channels') });
+            self.setState({channels: $(button).data('channels')});
         });
     },
     getInitialState: function() {
-        return { channels: [] };
+        return {channels: [], loadingPMChannel: -1};
```

Comment:
Oh so remove _all_ references to private message channels.  Okay, misunderstood, thought it was only stuff dealing with the backend since we call direct message channels private message channels to users, but it does make more sense to be completely consistent with variables and functions one way or the other, I'll change it when I get a chance tomorrow.